config.rb.sample

```diff
@@ -6,12 +9,17 @@ $new_discovery_url='https://discovery.etcd.io/new'
 #if File.exists?('user-data') && ARGV[0].eql?('up')
 #  require 'open-uri'
 #  require 'yaml'
-# 
+#
 #  token = open($new_discovery_url).read
-# 
+#
 #  data = YAML.load(IO.readlines('user-data')[1..-1].join)
-#  data['coreos']['etcd']['discovery'] = token
-# 
+#  if data['coreos'].key? 'etcd'
```

Comment:
if they delete the etcd section it will cause an error if we don't check, there is probably an idiomatic ruby sneaky way of doing this, but IANAR